torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -0,0 +1,224 @@
+import torch
+import itertools as it
+from typing import List, Optional, Dict
+from collections import namedtuple
+
+import torchaudio
+
+try:
+    torchaudio._extension._load_lib('libtorchaudio_decoder')
```

Comment:
Can you move the initialization of "libtorchaudio_decoder" to "__init.py__"?
Then wrap it with try except claus. After that in ctc_decoder.py, no need to try except the import statement of _torchaudio_decodrt